### PR TITLE
Added if statements to random message logs, only print if debugmode is true

### PR DESCRIPTION
**Pull request by Aalawfi**
Moved debugmode declaration to global scope and added if statements on each printf

### CODE REVIEW

File: j0lt.c [hunk 1]
```diff
@@ -654,12 +660,14 @@ print_hex(void* data, size_t len)
         size_t i, j;
         for (j = 0, i = 0; i < len; i++) {
                 if (i % 16 == 0) {
-                        printf("\n0x%.4zx: ", j);
```
Comment (thesc1ent1st) [inline j0lt.c:657]: Remove this if statement.

File: j0lt.c [hunk 2]
```diff
@@ -654,12 +660,14 @@ print_hex(void* data, size_t len)
...
+                        if(debugmode == true)
+                         printf("\n0x%.4zx: ", j);
                         j += 16;
                 }
                 if (i % 2 == 0)
                         putchar(' ');
-                printf("%.2x", d[i]);
+                if(debugmode == true)
```
Comment (7etsuo) [inline j0lt.c]: Also remove this if statement.

File: j0lt.c [hunk 3]
```diff
@@ -217,8 +219,8 @@ main(int argc, char** argv)
         posix_spawnattr_t* attrp;
         posix_spawn_file_actions_t file_actions;
         posix_spawn_file_actions_t* file_actionsp;
-
-        printf("%s", g_menu);
+        if(debugmode == true)
```
Comment (7etsuo) [inline j0lt.c]: remove this if statement.

File: j0lt.c [hunk 4]
```diff
@@ -654,12 +660,14 @@ print_hex(void* data, size_t len)
         size_t i, j;
         for (j = 0, i = 0; i < len; i++) {
                 if (i % 16 == 0) {
-                        printf("\n0x%.4zx: ", j);
+                        if(debugmode == true)
+                         printf("\n0x%.4zx: ", j);
```
Comment (7etsuo) [inline j0lt.c]: Remove this if statement

**Review by 7etsuo — CHANGES_REQUESTED** [review]
Hi @Aalawfi , I should have been a bit more specific with the change requests here.  I miss-spoke with the request. 

The only line that needs to be changed is as follows, 

if (debugmode) 
        printf("+ current attack magnitude %d \n", magnitude);

If you could please use tab sizes of 8 spaces, that would be great. 

Thank you again for taking this task on.

**Comment by Aalawfi** [comment]
Thank you for clarifying. This is my first contribution on GitHub. In the future, I will make sure to ask the right questions first. Thanks.

**Pull request merged**